### PR TITLE
Add (v)COW tokens for Ethereum Mainnet.

### PR DESCRIPTION
**Pull request by fairlighteth**
Adds the vCOW and COW tokens for Ethereum mainnet:

**Mainnet:**
Cow Token: [0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB](https://etherscan.io/address/0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB)
vCOW: [0xD057B63f5E69CF1B929b356b579Cba08D7688048](https://etherscan.io/token/0xD057B63f5E69CF1B929b356b579Cba08D7688048)

### CODE REVIEW

File: contract-map.json [hunk 1]
```diff
@@ -1,4 +1,32 @@
...
+  "0xc20C9C13E853fc64d054b73fF21d3636B2d97eaB": {
+    "name": "CoW Protocol Virtual Token",
+    "logo": "vCOW.png",
+    "erc20": true,
+    "symbol": "vCOW",
+    "decimals": 18
+  },
+  "0xD057B63f5E69CF1B929b356b579Cba08D7688048": {
+    "name": "CoW Protocol Virtual Token",
+    "logo": "vCOW.png",
```
Comment (MRabenda) [inline contract-map.json:11]: SVG is required

File: contract-map.json [hunk 2]
```diff
@@ -1,4 +1,32 @@
 {
+  "0xc20C9C13E853fc64d054b73fF21d3636B2d97eaB": {
```
Comment (MRabenda) [inline contract-map.json:2]: On which chain is deployed this contract?

Comment (fairlighteth) [reply contract-map.json:2]: > On which chain is deployed this contract?

As referenced in the PR description, this is on Gnosis Chain.

File: contract-map.json (hunk 1 above)
Comment (fairlighteth) [reply contract-map.json:11]: Is it? As I saw other token assets also using PNG. If it's a must we can create one.

File: contract-map.json (hunk 2 above)
Comment (MRabenda) [reply contract-map.json:2]: Can you separate contract on ETH from Gnosis to two separate PR?

Comment (fairlighteth) [reply contract-map.json]: @MRabenda Yes, removed Gnosis Chain for this PR.

**Review by MRabenda — APPROVED** [review]

**Pull request merged**